db/migrate/20190522230042_create_teachers.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTeachers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :teachers do |t|
+      t.string :first_name
+      t.string :last_name
+      t.string :email
+      t.string :password
```

Comment:
This looks to me like the `password` is being stored in plain text?

Reply:
I think has_secure_password takes care of this.